src/Raven.Server/Config/Categories/LogsConfiguration.cs

```diff
@@ -9,58 +10,78 @@ namespace Raven.Server.Config.Categories
     [ConfigurationCategory(ConfigurationCategoryType.Logs)]
     public sealed class LogsConfiguration : ConfigurationCategory
```

Comment:
In general I wonder if we can do more in terms of backward compatibility in those options.

Reply:
Let's talk tomorrow about this